examples/tornjak/README.md

```diff
@@ -3,20 +3,47 @@
 > [!Warning]
 > The current version of Tornjak in this chart is deployed without authentication. Therefore it is not suitable to run this version in production.
 
-To install Spire with the least privileges possible we deploy spire across 2 namespaces.
+## Deploy CRDs
+
+To install Tornjak with the least privileges possible we deploy SPIRE across 2 namespaces.
+Create the namespaces explicitly then deploy required CRDs:
 
 ```shell
 kubectl create namespace "spire-system"
 kubectl label namespace "spire-system" pod-security.kubernetes.io/enforce=privileged
 kubectl create namespace "spire-server"
 kubectl label namespace "spire-server" pod-security.kubernetes.io/enforce=restricted
```

Comment:
Now that we have the namespace creation bits in the chart, should we recommend here just using those?

Maybe though we should just recommend starting at the chart REAMDE.md instructions, getting it deployed, then coming back and`helm upgrad`ing the chart with the additional tornjak settings?

Reply:
Good suggestion. I switched the default install to use the namespace create and then referenced that from here